internal/hardware/retrieve.go

```diff
@@ -47,43 +48,42 @@ func (c *Client) Retrieve() *cobra.Command {
 			header := []string{"ID", "Facility", "Plan", "Created"}
 
 			inc := []string{}
+			exc := []string{}
+			var query, state, provisionable string
```

Comment:
Where do the values for these come from?  These could be wired up to the `Filters()` function you added

Reply:
updated